python/paddle/trainer/config_parser.py

```diff
@@ -686,12 +685,15 @@ def __init__(
         if num_filters is not None:
             self.operator_conf.num_filters = num_filters
 
-        parse_conv(conv_conf, input_layer_names[0], self.operator_conf.conv_conf, True)
+        parse_conv(conv_conf,
+                   MakeLayerNameInSubmodel(input_layer_names[0]),
```

Comment:
Only this place is needed. If you add MakeLayerNameInSubmodel to the base class, the Operator would be fine, but when feeding the operator to the input of the MixedLayer, the parent model name will be added twice.